ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -617,49 +617,58 @@ public class Manager: NSObject, UIGestureRecognizerDelegate {
       throw KMPError.wrongPackageType
     }
```

Comment:
Note that I paused development at this point, as the old version of `parseKbdKMP` (and later, `parseLMKMP`) are still reasonably recognizable, with logic mostly preserved.

Most of these methods' contents will be refactored out in the next PR, as noted by other comments.